src/vos/vos_meta.h

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Is this file needed?

Reply:
hmm, not sure what this is :-)...Probably a WIP that got added accidently.